src/js/components/user/info.tsx

```diff
@@ -285,6 +286,9 @@ const UserDropdown = () => {
                   >
                     {user.username}
                   </h2>
+                  <div className="slds-m-top_xx-small">
+                    <ManageAccountButton />
+                  </div>
```

Comment:
Hrm, functionally it's kind of annoying that clicking this link takes you to the Manage Account page, but does not automatically close this Popover. 🤔 We may want to make this a controlled component (using `onClick`, `onRequestClose`, and `isOpen` props) so that we can manually close the popover when this link is clicked.